build.sbt

```diff
@@ -61,7 +61,7 @@ lazy val updateNpm = taskKey[Unit]("Update npm")
 lazy val npmTask   = inputKey[Unit]("Run npm with arguments")
 
 lazy val commonSettings = Seq(
-  organization := "com.softwaremill",
+  organization := "softwaremill",
```

Comment:
why not `com.softwaremill`?

Reply:
Value of `organization` is used as part of docker image tag. (the "username"). By convention all docker images stored on docker hub are named `software mill/*`. 
I think `organization` is not used in other places, so why to have 2 values if we can have only one?

Reply:
The convention in Scala/Java is for the org nam eto be always reverse domain name. For libraries this specifies part of the coordinates (for Maven/SBT). So I'd prefer to keep this separate using `organization in Docker` or sth like that